test/BootstrapCommand.js

```diff
@@ -4,63 +4,179 @@ import path from "path";
 import fs from "fs";
 
 import ChildProcessUtilities from "../src/ChildProcessUtilities";
+import FileSystemUtilities from "../src/FileSystemUtilities";
 import BootstrapCommand from "../src/commands/BootstrapCommand";
 import exitWithCode from "./_exitWithCode";
 import initFixture from "./_initFixture";
+import stub from "./_stub";
 
 import assertStubbedCalls from "./_assertStubbedCalls";
 
 describe("BootstrapCommand", () => {
-  let testDir;
```

Comment:
The diff is cluttered here due to indentation changes (added a nested `describe` layer).

I've left the original test unchanged, and added three additional tests.